src/main/resources/lib/menu.js

```diff
@@ -120,37 +117,91 @@ exports.getMenuTree = function (levels) {
  * Returns submenus of a parent menuitem.
  * @param {Content} parentContent - content object obtained with 'portal.getContent', 'portal.getSite' or any 'content.*' commands
  * @param {Integer} levels - The number of submenus to retrieve
+ * @param {Object} params - parameteres to configure 
+ *   @param {String} [params.urlType=Server] - Control type of URL to be generated for menu items, default is 'server', only other option is 'absolute'.
  * @return {Array} Array of submenus
  */
-exports.getSubMenus = function (parentContent, levels) {
-    return doGetSubMenus(parentContent, levels);
-};
+exports.getSubMenus = getSubMenus;
+
+function getSubMenus(parentContent, levels, params = {}) {
+    levels = levels || 1;
+    const settings = {
+        urlType: params.urlType == "absolute" ? "absolute" : "server",
+    };
 
-var doGetSubMenus = function (parentContent, levels) {
-    var subMenus = [];
-    var currentContent = libs.portal.getContent() || parentContent; // Fallback to currentContent if there's no content (like in errorHandlers).;
+    // Fallback to currentContent if there's no content (like in errorHandlers).
+    var currentContent = libs.portal.getContent() || parentContent;
```

Comment:
This file uses both `var` and `let, const`. Once you start rewriting code to use let and const, you should change all declarations in that file to match the same convention before you commit.

Reply:
The var here is on purpose, since the variable is used outside of the function scope. 
It would probably still work, since the functions are nesten in eachother. 
I just wanted to signify that submethods use that variable. 
Replace with a comment and use let?